glib/src/subclass/types.rs

```diff
@@ -186,21 +186,41 @@ impl<U: IsClass + ParentClassIs> IsSubclassableExt for U {
     }
 }
 
+// FIXME: same thing as IsSubclassable
+#[doc(hidden)]
+pub trait IsImplementableDefault<T: ObjectSubclass>: IsInterface {
```

Comment:
The extra trait doesn't seem to be necessary; we can just make trait methods do nothing by default? :)

Reply:
Yrd